app/concepts/application_operation.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class ApplicationOperation
+  include Dry::Transaction
+
+  def self.call(params, &block)
+    params = params.permit!.to_h if params.is_a?(ActionController::Parameters)
```

Comment:
It's not right level. Operation can works with any type of object. Please remove it.

Reply:
Removed